shell/browser/electron_api_ipc_handler_impl.cc

```diff
@@ -69,6 +90,9 @@ void ElectronApiIPCHandlerImpl::MessageSync(bool internal,
                                             const std::string& channel,
                                             blink::CloneableMessage arguments,
                                             MessageSyncCallback callback) {
```

Comment:
should this callback also handle the case when the `api::WebContents` is not available ?